src/components/AttachmentPicker/checkCameraPermission/index.js

```diff
@@ -0,0 +1,6 @@
+export default function checkCameraPermission() {
+    /**
+    * No need to check for android so It return true
+    */
+    return true;
```

Comment:
Is this true? Android has the camera permission requirement too. Or do you mean that it is not necessary for some other reason?

Reply:
Before opening camera we are asking user for a permission of camera.

File: `src/components/AttachmentPicker/launchCamera.android.js`

Reply:
But is it not possible to align both iOS and Android here?

Reply:
@Julesssss We can only check, permission is granted or not for iOS. But yes let's take opinion from @mananjadhav

Reply:
@dharmik Quick question if first time we reject the permission in Android, and then try to access the camera again, what happens as of now with your current change/?

Reply:
@mananjadhav It will pass threw this permission file:
`src/components/AttachmentPicker/launchCamera.android.js` where code is to ask permission and check if it granted or not.